runtime/gc_modron_startup/mminit.cpp

```diff
@@ -2067,8 +2067,7 @@ combinationMemoryParameterVerification(J9JavaVM *javaVM, IDATA* memoryParameters
 				 * space to Xmos
 				 */
 				if ((candidateXmosValue + candidateXmnsValue) > maximumXmsValue) {
-					candidateXmnsValue = newSpaceSizeMinimum;
-					candidateXmosValue = maximumXmsValue - candidateXmnsValue;
+					candidateXmnsValue = maximumXmsValue - candidateXmosValue;
```

Comment:
this might be safer?
					candidateXmnsValue = max(newSpaceSizeMinimum, maximumXmsValue - candidateXmosValue);
					candidateXmosValue = maximumXmsValue - candidateXmnsValue;

Reply:
Done. Thanks